src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.cshtml

```diff
@@ -86,16 +86,21 @@
 
         <div id="stackpage" class="page">
             <ul>
-                @{ int tabIndex = 6; }
+                @{ int exceptionCount = 0; int stackFrameCount = 0; string exceptionDetailId = ""; string frameId = ""; }
```

Comment:
Can put this on multiple lines too. Not sure where this awful convention came from in this file...